src/nwb/NWBFile.hpp

```diff
@@ -134,9 +120,12 @@ class NWBFile
    * @brief Saves the specification files for the schema.
    * @param specPath The location in the file to store the spec information.
    * @param versionNumber The version number of the specification files.
+   * @param registry The registry of specification files.
```

Comment:
Can you add a note where the registry is coming from. Passing pointers to functions is fine but may not be intuitive to other developers so a bit more documentation about this would be useful. Maybe a brief comment here and then a bit more details in the developer docs discussing how updating the schema works.

Reply:
added a brief note and a page to the developer docs